templates/collectd.conf.j2

```diff
@@ -69,22 +71,19 @@ LoadPlugin syslog
 #	PrintSeverity false
 #</Plugin>
 
-#<Plugin log_logstash>
+{% if installed_version|version_compare("5.5", ">=") %}#<Plugin log_logstash>
 #	LogLevel info
 #	File "/var/log/collectd.json.log"
 #</Plugin>
-
-<Plugin syslog>
-	LogLevel info
-</Plugin>
+{% endif %}
 
 ##############################################################################
```

Comment:
Please remove all the commented configuration from that point.
It's unwanted noise and make it harder to maintain.